controllers/chat/ChatController.js

```diff
@@ -13,9 +13,133 @@ const BetterSocialConstantListUtils = require('../../services/bettersocial/const
 const {CHANNEL_TYPE} = require('../../helpers/constants');
 const ChatAnonUserInfoFunction = require('../../databases/functions/chatAnonUserInfo');
 const BetterSocialCore = require('../../services/bettersocial');
+const {Op} = require('sequelize');
+const crypto = require('crypto');
 
 const v = new Validator();
 
+const generate_channel_id_for_anon_chat = (owner, member) => {
+  const hash = crypto.createHash('sha256');
+  hash.update(owner, member);
+  return hash.digest('hex');
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This function generates a unique channel ID for an anonymous chat by hashing the owner and member IDs. It uses the `crypto` library's `createHash` method to create a SHA256 hash of the concatenated owner and member IDs, then returns the hexadecimal representation of the hash.
```javascript
const generate_channel_id_for_anon_chat = (owner, member) => {
  const hash = crypto.createHash('sha256');
  hash.update(owner, member);
  return hash.digest('hex');
};
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->